In [1]:
%%html
<!--Script block to left align Markdown Tables-->
<style>
  table {margin-left: 0 !important;}
</style>

# Worksheet for Currilog Entry
Below generates markdown formatted minimal Currilog Entry

Look up CIP codes here http://www.txhighereddata.org/Interactive/CIP/

In [6]:
course_prefix = str(input('course prefix'))
old_course_number = str(input('Old Course Number'))
new_course_number = str(input('New Course Number'))
change_type = str(input('Change Type: new course, change number, etc.'))
current_title = str(input('current title'))
change_title = str(input('change title'))
current_description = str(input('current description'))
change_description = str(input('change description'))
credit_hours = str(input('credit hours'))
cip_code = str(input('cip code'))
academic_justification = str(input('academic justify'))
resource_justification = str(input('resource justify'))

course prefix CE
Old Course Number 4361
New Course Number 3361
Change Type: new course, change number, etc. Change course from senior level to junior level course number
current title Transportation  Engineering 
change title Transportation  Engineering 
current description  Transportation modes;  railway  and  airport  runway  design;  basic  design  and  analysis  concepts of highway systems; transportation planning; traffic engineer-ing; intersection control; geometrics; pavement engineering. 
change description Transportation modes;  railway and airport runway design;basic design and analysis concepts of highway systems; transportation planning; traffic engineering; intersection control; geometrics; pavement engineering. 
credit hours 3
cip code 14.0804.0006 
academic justify Course relocated in curriculum so capstone course is not concurrent and students have design exposure in transportation for senior design project
resource justify current teaching resources are to be used, num

In [7]:
print('|Item|Value|')
print('|---|---|')
print('|Course Prefix|',course_prefix,'|')
print('|Old Course Number|',old_course_number,'|')
print('|New Course Number|',new_course_number,'|')
print('|Change Type|',change_type,'|')
print('|Current Title|',current_title,'|')
print('|Change Title|',change_title,'|')
print('|Current Description|',current_description,'|')
print('|Change Description|',change_description,'|')
print('|Credit Hours|',credit_hours,'|')
print('|Cip Code|',cip_code,'|')
print('|Academic Justification|',academic_justification,'|')
print('|Resource Justification|',resource_justification,'|')

|Item|Value|
|---|---|
|Course Prefix| CE |
|Old Course Number| 4361 |
|New Course Number| 3361 |
|Change Type| Change course from senior level to junior level course number |
|Current Title| Transportation  Engineering  |
|Change Title| Transportation  Engineering  |
|Current Description|  Transportation modes;  railway  and  airport  runway  design;  basic  design  and  analysis  concepts of highway systems; transportation planning; traffic engineer-ing; intersection control; geometrics; pavement engineering.  |
|Change Description| Transportation modes;  railway and airport runway design;basic design and analysis concepts of highway systems; transportation planning; traffic engineering; intersection control; geometrics; pavement engineering.  |
|Credit Hours| 3 |
|Cip Code| 14.0804.0006  |
|Academic Justification| Course relocated in curriculum so capstone course is not concurrent and students have design exposure in transportation for senior design project |
|Resource Justification

|Item|Value|
|---|---|
|Course Prefix| CE |
|Old Course Number| 4361 |
|New Course Number| 3361 |
|Change Type| Change course from senior level to junior level course number |
|Current Title| Transportation  Engineering  |
|Change Title| Transportation  Engineering  |
|Current Description|  Transportation modes;  railway  and  airport  runway  design;  basic  design  and  analysis  concepts of highway systems; transportation planning; traffic engineer-ing; intersection control; geometrics; pavement engineering.  |
|Change Description| Transportation modes;  railway and airport runway design;basic design and analysis concepts of highway systems; transportation planning; traffic engineering; intersection control; geometrics; pavement engineering.  |
|Credit Hours| 3 |
|Cip Code| 14.0804.0006  |
|Academic Justification| Course relocated in curriculum so capstone course is not concurrent and students have design exposure in transportation for senior design project |
|Resource Justification| current teaching resources are to be used, number change only |